### PR TITLE
[qtdeclarative] Add qtchooser dependency for tools package. Contribut…

### PR DESCRIPTION
**Pull request by blammit**
…es to JB#50383

### CODE REVIEW

File: rpm/qtdeclarative.spec [hunk 1]
```diff
@@ -112,6 +112,7 @@ Summary:    Qt Declarative QtQmlDevTools - development files
 Requires:   %{name} = %{version}-%{release}
 Requires:   %{name}-devel = %{version}-%{release}
 Requires:   qt5-qtdeclarative-devel = %{version}-%{release}
+Requires:   qtchooser
```
Comment (pvuorela) [inline rpm/qtdeclarative.spec]: qtdeclarativetools-devel? This doesn't look like the right subpackage.

Comment (blammit) [reply rpm/qtdeclarative.spec]: Is it devel-tools instead?

**Comment by pvuorela** [comment]
Should work better there. The bug reference is wrong, though, on the commit message. Lgtm with that fixed.  Could be also an option to add the dependency to qmlscene subpackage.

**Comment by blammit** [comment]
Added to qmlscene package as well.

**Review by pvuorela — APPROVED** [review]
Still good. Adding the official approval too.

**Pull request merged**